anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
Here's an example I have.  This note generates a bunch of cards.  With the current anki, every card will start reciting every sound from the beginning, regardless of the clozes which are displayed for the card. When I ask for the answer it it does it again.  Regardless of where the sound is, on the question or the answer. It recites all of them in order.

With my change,  each card only sounds the sounds for the clozes it owns.  Either for the question or the answer, or both, however you want it.   If you want audio outside of the cloze, that works too.

I believe that the behavior is that which you desire.

2015, la plus chaude des années, et de loin.

L’annonce était pressentie. Elle est {{c1::désormais[sound:pronunciation_fr_désormais.mp3]::à partir de Maintenant}} officielle. L’année 2015 a été – et de loin – la plus chaude de l’histoire moderne. L’Agence océanique et atmosphérique américaine (National Oceanic and Atmospheric Administration, NOAA) et l’Agence spatiale américaine (National Aeronautics and Space Administration, NASA), qui {{c2::tiennent[sound:pronunciation_fr_tiennent.mp3]::posséder}} {{c3::toutes les deux[sound:pronunciation_fr_toutes_les_deux.mp3]:: chacun des deux ensemble.}} le registre des températures de la planète, l’ont confirmé {{c4::conjointement[sound:pronunciation_fr_conjointement.mp3]:: ensemble ; simultanément.}}, mercredi 20 janvier.